tools/.style.yapf

```diff
@@ -0,0 +1,3 @@
+[style]
+based_on_style=Google
```

Comment:
👍 I'll add a reference to the styleguide.

I do think it would be useful to have a section of the style guide specifically for Python. I can work on writing something up and send it your way for review since you are likely much more familiar with the style preferences that already exist in the project.

I think for this first pull request it's a first pass at normalizing around an existing standard. Once we have a style guide for Python we can work on adding custom rules to override things such as single vs double quotes. I do find it odd that there is no enforcement around quotes. The project mentions `Be consistent with your choice of string quote character within a file. Pick ' or " and stick with it.` but does not allow for a style entry to enforce it.

Reply:
OK no worries for this PR. Maybe a TODO to tackle this at some point.